components/channel_view/channel_view.jsx

```diff
@@ -125,7 +137,29 @@ export default class ChannelView extends React.PureComponent {
         }
 
         let createPost;
-        if (this.props.deactivatedChannel) {
+        if (!this.props.teammate && this.props.channelId) {
```

Comment:
Do we need to handle this case separately from the deactivated case? What if we just add the "Remove from Sidebar" button to the deactivated user message and just use that. I don't think we really need an extra message and everything for this case since it's an edge case

Reply:
Makes sense, I will remove the deactivated user case and use the same for both. BTW i noticed it was a bug in the CLI command i will be creating a new ticket for it. This for now anyway fixes it and also for the affected users so far. Worst case this view can be used for the deactivated users.

Reply:
Inititaly i thought it was through mysql command and not though CLI my bad